mappings/net/minecraft/entity/ai/goal/AvoidGoal.mapping

```diff
@@ -1,3 +1,5 @@
 CLASS aof net/minecraft/entity/ai/goal/AvoidGoal
+	FIELD a AVOID_PREDICATE Lapp;
```

Comment:
Hmm, how does this work out? `AVOID` is meaningless here as the field is already in `AvoidGoal` class.